src/main/java/com/oltpbenchmark/DBWorkload.java

```diff
@@ -775,4 +791,40 @@ private static boolean isBooleanOptionSet(CommandLine argsLine, String key) {
     }
     return (false);
   }
+
+  /**
+   * Handles the anonymization of specified tables with differential privacy and automatically
+   * creates an anonymized copy of the table. Adapts templated query file if sensitive values are
+   * present
+   *
+   * @param xmlConfig
+   * @param configFile
+   */
+  private static void applyAnonymization(XMLConfiguration xmlConfig, String configFile) {
+
+    String templatesPath = "";
+    if (xmlConfig.containsKey("query_templates_file")) {
+      templatesPath = xmlConfig.getString("query_templates_file");
+    }
+
+    LOG.info("Starting the Anonymization process");
+    LOG.info(SINGLE_LINE);
+    ProcessBuilder processBuilder =
+        new ProcessBuilder(
+            "python3", "scripts/anonymization/anonymizer.py", configFile, templatesPath);
```

Comment:
Why python and not java?
Alt, why does this need to be invoked directly in benchbase at all?

For instance, the following shell could suffice, no?

```sh
java -jar benchbase.jar --load=true ...
anononymizer.py ...
java -jar benchbase.jar --execute=true ...
```

Reply:
Python because the algorithms are only available in Python. 

It does not need to be invoked directly (You can run it as a standalone) but we thought it would be nice to have since there is information from the config file of BenchBase the script re-uses (JDBC connection info)